src/node/desktop/src/main/desktop-options.ts

```diff
@@ -17,10 +17,49 @@
 import Store from 'electron-store';
 import { MainWindow } from './main-window';
  
+const kProportionalFont = 'Font.ProportionalFont';
+const kFixWidthFont = 'Font.FixWidthFont';
+const kUseFontConfigDb = 'Font.UseFontConfigDb';
+
+const kZoomLevel = 'View.ZoomLevel';
+const kWindowBounds = 'View.WindowBounds';
+const kAccessibility = 'View.Accessibility';
+
+const kLastRemoteSessionUrl = 'Session.LastRemoteSessionUrl';
+const kAuthCookies = 'Session.AuthCookies';
+const kTempAuthCookies = 'Session.TempAuthCookies';
+
+const kIgnoredUpdateVersions = 'General.IgnoredUpdateVersions';
+const kClipboardMonitoring = 'General.ClipboardMonitoring';
+
+const kRBinDir = 'WindowsOnly.RBinDir';
```

Comment:
bike-shedding: what about `Platform.Windows.RBinDir`? And in the future we could use `Platform.MacOS` or `Platform.Linux` for other OS-specific options.

Reply:
I like that significantly better, great idea!